File: PYTHON/PerangkatElektronik.py
class PerangkatElektronik:
    def __init__(self, merk='', nama='', tahun=0):
        self.merk = merk
        self.nama = nama
        self.tahun = tahun

    # Setter untuk merk
    def set_merk(self, merk):
        self.merk = merk

    # Getter untuk merk
    def get_merk(self):
        return self.merk

    # Setter untuk nama
    def set_nama(self, nama):
        self.nama = nama

    # Getter untuk nama
    def get_nama(self):
        return self.nama

    # Setter untuk tahun
    def set_tahun(self, tahun):
        self.tahun = tahun

    # Getter untuk tahun
    def get_tahun(self):
        return self.tahun

File: CPP/PerangkatElektronik.cpp
#pragma once
#include <iostream>
#include <string>

using namespace std;

class PerangkatElektronik {
protected:
    string merk;
    string nama;
    int tahun;

public:
    PerangkatElektronik() {}

    PerangkatElektronik(string merk, string nama, int tahun) {
        this->merk = merk;
        this->nama = nama;
        this->tahun = tahun;
    }

    void setMerk(string merk) {
        this->merk = merk;
    }

    string getMerk() {
        return merk;
    }

    void setNama(string nama) {
        this->nama = nama;
    }

    string getNama() {
        return nama;
    }

    void setTahun(int tahun) {
        this->tahun = tahun;
    }

    int getTahun() {
        return tahun;
    }

    virtual ~PerangkatElektronik() {}
};
